.eslintrc.json

```diff
@@ -113,7 +113,8 @@
                 "node": true
             },
             "rules": {
-                "no-console": 0
+                "no-console": 0,
+                "@typescript-eslint/no-undef": "off",
```

Comment:
Maybe this should be a more targeted exception, either just a comment in the file in question or declaring `backdoors` as a global for tests.

Reply:
I looked around a bunch for how to fix this "correctly" and this is the [advice I found](https://typescript-eslint.io/docs/linting/troubleshooting/#i-get-errors-from-the-no-undef-rule-about-global-variables-not-being-defined-even-though-there-are-no-typescript-errors):

> We strongly recommend that you do not use the `no-undef` lint rule on TypeScript projects. The checks it provides are already provided by TypeScript without the need for configuration - TypeScript just does this significantly better.

This is set up automatically if we use the configuration suggested by https://typescript-eslint.io/docs/linting/#configuration. Happy to do this in another change.